### PR TITLE
Added function to calculate Armenteros-Podolanski variables

### PR DESCRIPTION
**Pull request by eduardo-rodrigues**

### CODE REVIEW

File: skhep/math/kinematics.py [hunk 1]
```diff
@@ -47,6 +50,64 @@ def Kallen_function(x, y, z):
...
+    -------
+    Tuple :math:`(\\alpha,q_T)`.
+
+    References
+    ----------
+    .. [APPaper] J. Podolanski and R. Armenteros, III. Analysis of V-events, The London, Edinburgh, and Dublin Philosophical Magazine and Journal of Science 45 (1954) 13, http://dx.doi.org/10.1080/14786440108520416
+    """
+
+    mother_mag = (pplus_3Dvec+pminus_3Dvec).mag
+    if isequal (mother_mag, 0.0):
```
Comment (chrisburr) [inline skhep/math/kinematics.py]: `if isequal (mother_mag, 0.0):` → `if isequal(mother_mag, 0.0):`

File: skhep/math/kinematics.py [hunk 2]
```diff
@@ -47,6 +50,64 @@ def Kallen_function(x, y, z):
...
+        raise ValueError( 'Total momentum has zero magnitude!')
```
Comment (chrisburr) [inline skhep/math/kinematics.py]: `raise ValueError( 'Total momentum has zero magnitude!')` → `raise ValueError('Total momentum has zero magnitude!')`

File: skhep/math/kinematics.py [hunk 3]
```diff
@@ -47,6 +50,64 @@ def Kallen_function(x, y, z):
...
+
+    # Longitudinal momentum asymmetry, i.e. imbalance
+    alpha = (pplus_3Dvec.mag2-pminus_3Dvec.mag2)/mother_mag**2
+
+    # Transverse momentum of positively-charged particle along the mother particle momentum direction
+    qT = (pplus_3Dvec.cross(pminus_3Dvec)).mag/mother_mag
+
+    return (qT, alpha)
```
Comment (chrisburr) [inline skhep/math/kinematics.py]: `return (qT, alpha)` → `return qT, alpha`
This one I feel less strongly about, though, the only other cases of multiple return values in scikit-hep are in the tests and don't use parentheses to return multiple values.

File: skhep/math/kinematics.py [hunk 4]
```diff
@@ -47,6 +50,64 @@ def Kallen_function(x, y, z):
     """
     return (x - y - z) ** 2 - 4 * y * z  # faster to evaluate condensed form used
 
+def Armenteros_Podolanski_variables(pplus_3Dvec, pminus_3Dvec):
```
Comment (chrisburr) [inline skhep/math/kinematics.py]: [There should be two black lines between top level functions](https://www.python.org/dev/peps/pep-0008/#blank-lines) like the other functions in this file.

**Comment by eduardo-rodrigues** [comment]
I'm amazed that I did not catch most if not all of these! Indeed I totally agree with you. I will fix this asap. Thanks.

File: skhep/math/kinematics.py (hunk 3 above)
Comment (eduardo-rodrigues) [reply skhep/math/kinematics.py]: I would rather "fix" the other cases. I kind of like the idea of being explicit about the tuple being returned. But no strong feeling.

**Comment by eduardo-rodrigues** [comment]
I thought about this a bit more, @VanyaBelyaev. I tend to agree that it's the natural argument but on the other hand it does require a bit more knowledge about the internals of the n-vector classes. By passing the 3-vector straightaway all kinds of 3-vectors will work provided they respond to just 2-3 methods, which are reasonably standard.

There is also the fact that we are rediscussing at this stage a new NumPy-based set of vector classes, so this may be revisited anyway once we converge there.

With this in mind, I would prefer to go with the present code *for now*. We will come back to this once we converge on the more performant classes.

OK if we then merge this? Thanks.

**Pull request merged**